models/labels/addresses/infrastructure/identifier/burn_addresses/labels_burn_addresses.sql

```diff
@@ -1,45 +1,44 @@
-    , ('ethereum', '0x4444444444444444444444444444444444444444', 'Burn Address', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-26'), now(), 'burn_addresses', 'identifier')
-    , ('ethereum', '0x5555555555555555555555555555555555555555', 'Burn Address', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-26'), now(), 'burn_addresses', 'identifier')
-    , ('ethereum', '0x6666666666666666666666666666666666666666', 'Burn Address', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-26'), now(), 'burn_addresses', 'identifier')
-    , ('ethereum', '0x7777777777777777777777777777777777777777', 'Burn Address', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-26'), now(), 'burn_addresses', 'identifier')
-    , ('ethereum', '0x8888888888888888888888888888888888888888', 'Burn Address', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-26'), now(), 'burn_addresses', 'identifier')
-    , ('ethereum', '0x9999999999999999999999999999999999999999', 'Burn Address', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-26'), now(), 'burn_addresses', 'identifier')
-    , ('ethereum', '0xbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb', 'Burn Address', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-26'), now(), 'burn_addresses', 'identifier')
-    , ('ethereum', '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee', 'Burn Address', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-26'), now(), 'burn_addresses', 'identifier')
-    , ('ethereum', '0xffffffffffffffffffffffffffffffffffffffff', 'Burn Address', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-26'), now(), 'burn_addresses', 'identifier')
-    , ('ethereum', '0x000000000000000000000000000000000000dead', 'Burn Address', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-26'), now(), 'burn_addresses', 'identifier')
-    , ('ethereum', '0x00000000000000000000045261d4ee77acdb3286', 'Burn Address', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-26'), now(), 'burn_addresses', 'identifier')
-    , ('ethereum', '0xdead000000000000000042069420694206942069', 'Burn Address', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-26'), now(), 'burn_addresses', 'identifier')
-    , ('ethereum', '0x0123456789012345678901234567890123456789', 'Burn Address', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-26'), now(), 'burn_addresses', 'identifier')
-    , ('ethereum', '0x1234567890123456789012345678901234567890', 'Burn Address', 'infrastructure', 'hildobby', 'static', timestamp('2023-03-26'), now(), 'burn_addresses', 'identifier')
+    ('ethereum', 0x0000000000000000000000000000000000000000, 'Burn Address', 'infrastructure', 'hildobby', 'static', DATE '2023-03-26' , now(), 'burn_addresses', 'identifier')
+    , ('ethereum', 0x0000000000000000000000000000000000000001, 'Burn Address', 'infrastructure', 'hildobby', 'static', DATE '2023-03-26' , now(), 'burn_addresses', 'identifier')
```

Comment:
Yes, the data type may change.
I check the document, `DATE` type is a subclass of `TIMESTAMP` type.
https://trino.io/docs/current/language/types.html#timestamp-p
It may still be compatible, but not rigorous enough.
I am still searching for a perfect solution (and this is one issue I want to raise to the team).

Reply:
i'm seeing this function usage to get timestamp to work on write:
https://github.com/duneanalytics/spellbook/pull/3620/files#diff-121b9dd26a1080c036af11dda045f34652fce07dc6751977fce8516c119b0bebR35

can we use that?

Reply:
ignore, we have `timestamp` available now